src/Worker/PluginExtractor.php

```diff
@@ -66,7 +66,7 @@ public function extract($sourceFile)
         // Get plugins key to return
         // @TODO Verify that this works with lots of plugins (but it should...)
         $stat = $zipArchive->statIndex(0);
-        $folderName = trim($stat['name'], '/');
```

Comment:
Looks like it is possible in ZIP files to skip creating top-level folders... `Frontend/PluginName` seems to be totally legit...

Reply:
Yeah, I think this saves some space in the meta-data :)

I will add some tests within a later pull-request to ensure this works as expected every time and makes it easier to add new situations